assets/static/css/reset.css

```diff
@@ -0,0 +1,25 @@
+html {
+    box-sizing: border-box;
+    font-size: 16px;
+}
+  
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+  
+body, p, ol, ul {
+    margin: 0;
+    padding: 0;
+    font-weight: normal;
+    padding: 3.125rem 0;
+    /* font: normal 1rem/1.5 "Fira Sans", "Helvetica Neue", sans-serif; */
+}
+  
+ol, ul {
+    list-style: none;
+}
+  
+img {
+    max-width: 100%;
+    height: auto;
+}
```

Comment:
Need a newline EOF

Reply:
sorry i had the default VS code settings, fixed